src/psyclone/psyir/nodes/psy_data_node.py

```diff
@@ -81,17 +81,17 @@ class PSyDataNode(Statement):
     :type ast: sub-class of :py:class:`fparser.two.Fortran2003.Base`
     :param children: the PSyIR nodes that are children of this node. These \
         will be made children of the child Schedule of this PSyDataNode.
-    :type children: List[:py:class:`psyclone.psyir.nodes.Node`]
+    :type children: list[:py:class:`psyclone.psyir.nodes.Node`]
     :param parent: the parent of this node in the PSyIR tree.
     :type parent: :py:class:`psyclone.psyir.nodes.Node`
-    :param options: a dictionary with options for transformations.
-    :type options: Optional[Dict[str, Any]]
+    :param options: a dict[ionary with options for transformations.
```

Comment:
Your search and replace was a bit too keen here...

Reply:
Oops - in many places :( I believe I have fixed them all.